conda/conda-build/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "legate-core" %}
+{% if package_version is defined %}
+    {% set version = package_version %}
+{% elif 'dev' in environ.get('GIT_DESCRIBE_TAG', '') %}
+    {% set version = environ.get('GIT_DESCRIBE_TAG', '') ~ environ.get('GIT_DESCRIBE_NUMBER', '') %}
```

Comment:
Can you add a comment here describing the expected format for the tag?

Reply:
Done. I think that we should have more detail on the workflow in a separate document, but here at least we can say what tags are expected.